l5kit/l5kit/data/zarr_dataset.py

```diff
@@ -37,7 +40,13 @@
     ("yaw", np.float32),
     ("velocity", np.float32, (2,)),
     ("track_id", np.uint64),
-    ("label_probabilities", np.float32, (len(LABELS),)),
+    ("label_probabilities", np.float32, (len(PERCEPTION_LABELS),)),
+]
+
+TL_FACE_DTYPE = [
+    ("gid", "<U16"),
```

Comment:
Minor: Don't like this naming here - this is the identifier for the traffic light? Should we make it comparable to agents, ie track_id = tl_id?
What is traffic_light_gid then btw? I do not see it used.

Reply:
> track_id = tl_id?

@yye-lyft are you fine with `tl_id` ?

> What is traffic_light_gid then btw? I do not see it used.


future usages and debug

Reply:
@lucabergamini I would not change it, because it is from protobuf

Reply:
@lucabergamini `gid ->face_id` and `traffic_light_gid -> traffic_light_id`?
@oliver-scheel `gid` refers to the traffic light face id and `traffic_light_gid` refers to the id of traffic light which can have multiple faces

Reply:
@lucabergamini similar to `frame -> agent`, where same frame can have multiple agents. Can we have `traffic_light -> traffic_light_faces`? Is it a better design?

Reply:
I don't think it is, but I'll ask @johnhoustonlyft